drivers/docker/config.go

```diff
@@ -134,6 +134,21 @@ var (
 		Name:              pluginName,
 	}
 
+	danglingContainersBlock = hclspec.NewObject(map[string]*hclspec.Spec{
+		"enabled": hclspec.NewDefault(
+			hclspec.NewAttr("enabled", "bool", false),
+			hclspec.NewLiteral(`true`),
+		),
+		"period": hclspec.NewDefault(
+			hclspec.NewAttr("period", "string", false),
+			hclspec.NewLiteral(`"5m"`),
+		),
+		"creation_timeout": hclspec.NewDefault(
+			hclspec.NewAttr("creation_timeout", "string", false),
```

Comment:
`grace` might be a more clear name as we use it in the [check restart stanza](https://www.nomadproject.io/docs/job-specification/check_restart.html#grace)

The "timeout" in `creation_timeout` just makes me think this has to do with API timeouts, not a grace period.